sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -148,7 +183,22 @@ def plot(self, ax=None, *, name=None, **kwargs):
             line_kwargs["label"] = name
         line_kwargs.update(**kwargs)
 
+        # If prevalence_pos_label is not provided, even if plot_chance_level
+        # is set to True, we do not plot the chance level line
+        if self.prevalence_pos_label is None:
+            plot_chance_level = False
+
+        if plot_chance_level:
```

Comment:
Let's move this definition of the kwargs inside the `if plot_chance_level` below to avoid having twice the `if` statement